testutils/resourcebuilder/cluster-api/infrastructure/v1beta2/powervscluster.go

```diff
@@ -0,0 +1,179 @@
+		},
+	}
+}
+
+// Object meta fields.
+
+// WithAnnotations sets the annotations for the IBMPowerVSCluster builder.
+func (p PowerVSClusterBuilder) WithAnnotations(annotations map[string]string) PowerVSClusterBuilder {
+	p.annotations = annotations
+	return p
+}
+
+// WithCreationTimestamp sets the creationTimestamp for the IBMPowerVSCluster builder.
+func (p PowerVSClusterBuilder) WithCreationTimestamp(timestamp metav1.Time) PowerVSClusterBuilder {
+	p.creationTimestamp = timestamp
+	return p
+}
+
+// WithDeletionTimestamp sets the deletionTimestamp for the IBMPowerVSCluster builder.
+func (p PowerVSClusterBuilder) WithDeletionTimestamp(timestamp *metav1.Time) PowerVSClusterBuilder {
+	p.deletionTimestamp = timestamp
+	return p
+}
+
+// WithGenerateName sets the generateName for the IBMPowerVSCluster builder.
+func (p PowerVSClusterBuilder) WithGenerateName(generateName string) PowerVSClusterBuilder {
+	p.generateName = generateName
+	return p
+}
+
+// WithLabels sets the labels for the IBMPowerVSCluster builder.
+func (p PowerVSClusterBuilder) WithLabels(labels map[string]string) PowerVSClusterBuilder {
+	p.labels = labels
+	return p
+}
+
+// WithName sets the name for the IBMPowerVSCluster builder.
+func (p PowerVSClusterBuilder) WithName(name string) PowerVSClusterBuilder {
+	p.name = name
+	return p
+}
+
+// WithNamespace sets the namespace for the IBMPowerVSCluster builder.
+func (p PowerVSClusterBuilder) WithNamespace(namespace string) PowerVSClusterBuilder {
+	p.namespace = namespace
+	return p
+}
+
+// Spec fields.
+
+// WithNetwork sets the network for the IBMPowerVSCluster builder.
+func (p PowerVSClusterBuilder) WithNetwork(network capibmv1.IBMPowerVSResourceReference) PowerVSClusterBuilder {
+	p.network = network
+	return p
+}
+
+// WithControlPlaneEndpoint sets the controlPlaneEndpoint for the IBMPowerVSCluster builder.
+func (p PowerVSClusterBuilder) WithControlPlaneEndpoint(endpoint clusterv1.APIEndpoint) PowerVSClusterBuilder {
+	p.controlPlaneEndpoint = endpoint
+	return p
+}
+
+// WithZone sets the zone for the IBMPowerVSCluster builder.
+func (p PowerVSClusterBuilder) WithZone(zone *string) PowerVSClusterBuilder {
+	p.zone = zone
+	return p
+}
+
+// WithServiceInstance sets the serviceInstance for the IBMPowerVSCluster builder.
+func (p PowerVSClusterBuilder) WithServiceInstance(serviceInstance *capibmv1.IBMPowerVSResourceReference) PowerVSClusterBuilder {
+	p.serviceInstance = serviceInstance
+	return p
+}
+
+// WithResourceGroup sets the resourceGroup for the IBMPowerVSCluster builder.
+func (p PowerVSClusterBuilder) WithResourceGroup(resourceGroup *capibmv1.IBMPowerVSResourceReference) PowerVSClusterBuilder {
+	p.resourceGroup = resourceGroup
+	return p
+}
+
+// WithLoadBalancer sets the loadBalancers for the IBMPowerVSCluster builder.
+func (p PowerVSClusterBuilder) WithLoadBalancer(loadBalancers []capibmv1.VPCLoadBalancerSpec) PowerVSClusterBuilder {
+	p.loadBalancers = loadBalancers
+	return p
+}
```

Comment:
Can you sort the fields alphabetically within each group, this should then be consistent with other providers